### ISSUE TITLE
Missing interactions in v3.0

### ISSUE THREAD

**Issue by kcotto** [griffithlab/dgi-db#231]
I initially noticed that we were missing interactions based on comparing numbers from v2.0 to v3.0. Specifically, I noticed that Clearity Foundation: Biomarkers, Clearity Foundation: Clinical Trials, GTP: Interactions, My Cancer Genome: Clinical Trials, and The Druggable Genome: Clinical Trials were down in their interaction_claims_count by  1%, 58%, 7%, 83%, 16% respectively.

So through digging through the commit logs, it looks like some claims were initially lost with data subrepo commit
```commit 8c90d5ff54a624b5fc923c0a1df9dcc0512cf745
Author: Alex H. Wagner <awagner@genome.wustl.edu>
Date:   Wed Jul 27 21:43:24 2016 -0500

    dump of local db after import and grouping```

However, it seems that most of the missing claims become absent in the data.sql file following the data subrepo commit
```commit 003505559bdc699204681d54ebdcdd3e2e35ba6f
Author: Susanna Kiwala <ssiebert@genome.wustl.edu>
Date:   Fri Mar 10 11:43:17 2017 -0600

    Dump of local db after creating interactions```

Looking back on that date (March 10th), that is when PR#126 was merged which is related to the interaction table and the interaction grouper.

The good news is that we have all of the interaction TSVs for these sources except The Druggable Genome: Clinical Trials. For the sources we have TSVs for, we will update these sources for v3.0. 

(GTP: Interactions will not be updated through this issue as an online importer is in the works in issue #148)

**Comment by ahwagner** [comment]
Branch [backup-v2](https://github.com/griffithlab/dgi-db/tree/backup-v2) adds functionality to restore any source from DGIdb v2. On doing so, I noted that the observed reduction in interaction claims is due to the database constraints created by @susannasiebert requiring interaction claims to be unique. I have restored each of the aforementioned sources, and found that our counts from each are consistent with our current, live database after collapsing redundant interaction claims.

**Comment by ahwagner** [comment]
closed in 414e6bfdd250630f772a766853e2b4c0714bbada
